politeiawww/user/user.go

```diff
@@ -388,12 +434,21 @@ type Database interface {
 	// Return user record given its id
 	UserGetById(uuid.UUID) (*User, error)
 
+	// Return user session record given its id
+	SessionGetById(uuid.UUID) (*Session, error)
+
 	// Return user record given a public key
 	UserGetByPubKey(string) (*User, error)
 
 	// Return a map of public key to user record
 	UsersGetByPubKey(pubKeys []string) (map[string]User, error)
 
+	// Delete the session with the given id
+	SessionDeleteById(uuid.UUID) error
```

Comment:
If a user has multiple sessions, presumably because they are logged in on multiple devices, how do you know that this is deleting the correct session?

Reply:
I assumed that each device/endpoint would have a single session only. The cookie (from the Gorilla session store) has a `sessionid` property that points to that `Session`. That's how we know which session to delete e.g. when the user logs out.

Reply:
oh right, I was thinking the UUID param was referring to the user ID, but its referring to the session ID.  Makes sense now.